mobile/android/android-emulator-services.ts

```diff
@@ -224,19 +226,26 @@ class AndroidEmulatorServices implements Mobile.IEmulatorPlatformServices {
 
 			// have to start new emulator
 			this.$logger.info("Starting Android emulator with image %s", image);
+			let emulatorProc: child_process.ChildProcess = null;
+			let emulatorProcEnded = false;
 			if(this.$options.geny) {
 				//player is part of Genymotion, it should be part of the PATH.
-				this.$childProcess.spawn("player", ["--vm-name", image],
-					{ stdio: "ignore", detached: true }).unref();
+				emulatorProc = this.$childProcess.spawn("player", ["--vm-name", image],
+					{ stdio: "ignore", detached: true });
 			} else {
-				this.$childProcess.spawn('emulator', ['-avd', image],
-					{ stdio: "ignore", detached: true }).unref();
+				emulatorProc = this.$childProcess.spawn('emulator', ['-avd', image],
+					{ stdio: "ignore", detached: true });
 			}
+			emulatorProc.on("error", () => emulatorProcEnded = true);
+			emulatorProc.on("exit", () => emulatorProcEnded = true);
```

Comment:
I found an issue when starting native android emulator - we receive exit event during its start, so we exit the code :(